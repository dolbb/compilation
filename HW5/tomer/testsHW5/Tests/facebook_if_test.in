void main()
{
	int a=50;
	int jg=17;
	if(jg>a and jg>10)
	{
		print("jg=");
		printi(jg);
		
	}else{
		print("2a=");
		printi(2*a);
	}
	print("\n");

}
