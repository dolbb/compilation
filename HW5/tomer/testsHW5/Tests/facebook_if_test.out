Loaded: ./exceptions.s
2a=100
